package.json

```
@@ -73,10 +73,11 @@
     "eslint": "eslint --fix '*.{js,json}' '{src,tests,benchmarks}/**/*.{ts,tsx}'",
     "eslint:ci": "eslint '*.{js,json}' '{src,tests,benchmarks}/**/*.{ts,tsx}'",
     "pretest": "tsc --noEmit",
-    "test": "yarn node --experimental-vm-modules $(yarn bin jest)",
-    "test:ci": "yarn node --experimental-vm-modules $(yarn bin jest)",
-    "test:dev": "yarn node --experimental-vm-modules $(yarn bin jest) --watch --no-coverage",
-    "test:coverage:watch": "yarn node --experimental-vm-modules $(yarn bin jest) --watch",
+    "test": "vitest",
+    "test:ui": "yarn run test --ui",
+    "test:coverage": "yarn run test run --coverage",
+    "test:ci": "yarn run test run",
+    "test:coverage:watch": "yarn run test --coverage",
```

Let's remove yarn dependency (which I added). I want to simplify scripts.
How about this?
```suggestion
    "test": "vitest --ui --coverage",
    "test:ci": "vitest",
```